chia/_tests/environments/wallet.py

```diff
@@ -26,6 +28,22 @@
 from chia.wallet.wallet_node_api import WalletNodeAPI
 from chia.wallet.wallet_state_manager import WalletStateManager
 
+STANDARD_TX_ENDPOINT_ARGS: dict[str, Any] = TransactionEndpoint(
+    rpc_info=NeedsWalletRPC(client_info=None, wallet_rpc_port=None, fingerprint=None),
+    tx_config_loader=NeedsTXConfig(
+        min_coin_amount=cli_amount_none,
+        max_coin_amount=cli_amount_none,
+        coins_to_exclude=(),
+        amounts_to_exclude=(),
+        reuse=None,
+    ),
+    transaction_writer=TransactionsOut(transaction_file_out=None),
+    fee=uint64(0),
+    push=True,
+    valid_at=None,
+    expires_at=None,
+).__dict__
```

Comment:
The `.__dict__` isn't quite as related to the constructor as the result of `dataclasses.asdict()` is.  `.asdict()` itself isn't even fully correct depending if you use leading `_`'s, but it's better.

Reply:
I forgot to explain this, but dataclasses.asdict is recursive.  Which will not work in the way I need it to.  I just need the top layer dict-ified.